.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        component: [docker, docs, lint]
+        component: [docker, lint]
```

Comment:
I have removed this from here, because RTD will build and preview the docs now I have modified the webhooks to trigger it.